main.py

```diff
@@ -1,5 +1,13 @@
 
 from trafficcloud import api
+import sys
 
 if __name__ == "__main__":
-    api.runTrajectoryAnalysis('OlinShorter')
+    if len(sys.argv) <= 1:
+        print('Please pass the name of the folder in project_dir that you would like to run analysis on.')
+    else:
+        project = sys.argv[1]
+        if 'project_dir/' in project:
```

Comment:
I thought of a more generalized way to handle the path to the project identifier esp. if the path is an absolute path.

```python
# assume project = project_dir/uuid or project = absolute/path/to/project_dir/uuid
project = project.strip('/') # make sure cases like project_dir/uuid/ doesn't mess us up
project_identifier = project.split('/')[-1] # take just the project_identifier
```